ludwig/models/trainer.py

```diff
@@ -569,10 +538,10 @@ def _is_valid_batch_size(batch_size):
                 low = batch_size
                 prev_batch_size = batch_size
                 try:
-                    # re-initialize model...
-                    model = LudwigModel.create_model(config, random_seed)
-                    self.train_for_tuning(
-                        model, training_set, batch_size, total_steps=3)
+                    # re-initalize model...
```

Comment:
nit: initialized

Reply:
I ended up removing this comment since we don't re-initialize the model here with the latest changes.